release/release_gold.sh

```diff
@@ -5,8 +5,33 @@ source _product_info_json.sh
 source _promotion.sh
 source _publishing.sh
 
+RELEASE_PROPERTIES_FILE="release-data/release.properties"
+LIFERAY_RELEASE_FORK_NAME=${LIFERAY_RELEASE_FORK_NAME:="liferay/liferay-portal.ee"}
+
+function tag_release {
+	if ( ! curl --data "{
+			\"message\":\"\",
+			\"object\":\"$(lc_get_property ${RELEASE_PROPERTIES_FILE} git.hash.liferay-portal-ee)\",
+			\"tag\":\"${LIFERAY_RELEASE_VERSION}\",
+			\"type\":\"commit\"
+			}" \
+		--fail \
+		--header "Accept: application/vnd.github+json" \
+		--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
+		--header "X-GitHub-Api-Version: 2022-11-28" \
+		--location https://api.github.com/repos/"${LIFERAY_RELEASE_FORK_NAME}"/git/tags \
+		--request POST \
+		--silent
+		)
+	then
+		lc_log ERROR "The requested URL could not be accessed."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	fi
+}
+
 function check_usage {
-	if [ -z "${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" ] || [ -z "${LIFERAY_RELEASE_VERSION}" ]
+	if [ -z "${LIFERAY_RELEASE_GITHUB_PAT}" ] || [ -z "${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" ] || [ -z "${LIFERAY_RELEASE_VERSION}" ]
```

Comment:
I changed my mind. IMO we should remove this and  exit the tag_release function with SKIPPED return code if it's not defined.